src/objective-c/GRPCClient/GRPCCall+Cronet.m

```diff
@@ -38,9 +38,8 @@
 
 @implementation GRPCCall (Cronet)
 
-+ (void)setUseCronet:(BOOL)useCronet
-                    :(cronet_engine *)cronet_engine {
-  use_cronet = useCronet;
++ (void)useCronet:(cronet_engine *)cronet_engine {
+  use_cronet = YES;
   g_cronet_engine = cronet_engine;
```

Comment:
nit: variables and method parameters should be named in camelCase, per [the style guide](https://google.github.io/styleguide/objcguide.xml?showone=Variable_Names#Variable_Names), and favor readability over shortness. So these should be `useCronet` and `globalCronetEngine`.